skyvern/forge/sdk/routes/internal_auth.py

```
@@ -0,0 +1,126 @@
+
+
+def _require_local_access(request: Request) -> None:
+    if settings.ENV != "local":
+        raise HTTPException(status.HTTP_403_FORBIDDEN, "Endpoint only available in local env")
+    if not _is_local_request(request):
+        raise HTTPException(status.HTTP_403_FORBIDDEN, "Endpoint requires localhost access")
+
+
+async def _evaluate_local_api_key(token: str) -> DiagnosticsResult:
+    token_candidate = token.strip()
+    if not token_candidate or token_candidate == "YOUR_API_KEY":
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "token_candidate"

(S105)